nuxt.config.ts

```diff
@@ -2,18 +2,19 @@ import { defineNuxtConfig } from 'nuxt3'
 
 // https://v3.nuxtjs.org/docs/directory-structure/nuxt.config
 export default defineNuxtConfig({
-  css: [],
+  css: ['~/assets/scss/main'],
```

Comment:
In the future, I'd like to set up the css and the scss like we do in Gothamist so we don't have a 5 mile css list in the inspector